frontend/src/Components/Post/Post.js

```diff
@@ -306,7 +306,7 @@ export default function Post({ handleDeletePost, post_initializer }) {
                   }}
                   show={showModal}
                 >
-                  {modalText}
+                  <A href={`/user/${post.user_id}`}> {modalText}</A>
```

Comment:
The A tag here is unnecessary.. A simple `{modalText}` would do the work.
```suggestion
                   {modalText}
```